package.json

```diff
@@ -12,7 +12,7 @@
     "build": "tsc",
     "prepublish": "npm run test",
     "test": "npm run build && npm run lint && mocha --ui tdd",
-    "format": "./node_modules/.bin/clang-format --style=file -i dom5.ts",
+    "format": "./node_modules/.bin/clang-format --style=file -i dom5.ts test/dom5_test.ts",
```

Comment:
We should pin to the exact version of clang-format to use. I think 1.0.49 is the latest

Mind pinning,reinstalling, and rerunning clang-format?